tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -714,6 +807,15 @@ public static void exit() {
         Platform.exit();
     }
 
+    @After
+    public void cleanup() {
+        Util.runAndWait(() -> {
+            robot.mouseRelease(MouseButton.PRIMARY, MouseButton.SECONDARY, MouseButton.MIDDLE,
+                    MouseButton.BACK, MouseButton.FORWARD);
```

Comment:
I was doing some final testing, and discovered that unconditionally releasing all buttons is causing problems on Windows. It seems that releasing a button over a window or the desktop background is causing an action to occur, which causes random menus to appear and the tests to fail. This suggests that a cleanup method that releases all buttons, even after tests that don't press any buttons, isn't what we want to do.

So it looks like I led you astray when I suggested doing this (I hadn't tested it on Windows after you made this change at my request). For now, I recommend either going back to the approach of releasing the buttons in those tests that do a corresponding mousePress, or else setting a variable indicating which button(s) need to be released by the `@After` clean-up method.